### PR TITLE
add poetry

### PR DESCRIPTION
**Pull request by jchate6**
Adds Poetry

Tests won't pass until tomtoolkit/tom_base#874 is released.

### CODE REVIEW

**Comment by JulienPeloton** [comment]
Thanks @jchate6 for the transition to `poetry`!

**Review by phycodurus — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+[tool.poetry]
+name = "tom-fink"
+version = "0.0.0"  # version supplied by poetry-dynamic-versioning
+description = "The Fink Alert Broker Module for the TOM Toolkit. Fink is a broker currently connected to ZTF. More information on Fink at https://fi"
```
Comment (phycodurus) [inline pyproject.toml:4]: typo here in url?

File: tom_fink/fink.py [hunk 2]
```diff
@@ -140,7 +142,7 @@ class FinkQueryForm(GenericQueryForm):
     )
 
     help_ssosearch = """
-    The list of arguments for retrieving SSO data can be found at {}/api/v1/sso.
+    The list of arguments for retrieving SSO data can be found through {}/api/v1/sso.
```
Comment (phycodurus) [inline tom_fink/fink.py:145]: this should be an f-string instead of using `.format()` 20-lines below.

**Review by phycodurus — CHANGES_REQUESTED** [review]

**Review by phycodurus — APPROVED** [review]
thanks!

**Pull request merged**